### PR TITLE
fix: width of widget and network balances on token details

### PR DESCRIPTION
**Pull request by kayleegeorge**
fix: widget width and network balances align
Screen Shot 2022-08-17 at 2 46 01 PM

### CODE REVIEW

File: src/components/Tokens/TokenDetails/BalanceSummary.tsx [hunk 1]
```diff
@@ -9,15 +9,14 @@ import styled, { useTheme } from 'styled-components/macro'
 import NetworkBalance from './NetworkBalance'
 
 const BalancesCard = styled.div`
-  width: 284px;
+  width: 320px;
```
Comment (vm) [inline src/components/Tokens/TokenDetails/BalanceSummary.tsx:12]: think you can make a container that wraps the balances and widget, set the width to 320px, then do width 100% for all items inside

Comment (kayleegeorge) [reply src/components/Tokens/TokenDetails/BalanceSummary.tsx:12]: is that necessary tho

Comment (kayleegeorge) [reply src/components/Tokens/TokenDetails/BalanceSummary.tsx:12]: bc i think widget you specify the height

Comment (kayleegeorge) [reply src/components/Tokens/TokenDetails/BalanceSummary.tsx:12]: meh ok ill change

File: src/pages/TokenDetails/index.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ const RightPanel = styled.div`
   display: flex;
   flex-direction: column;
   gap: 20px;
+  width: 320px;
```
Comment (vm) [inline src/pages/TokenDetails/index.tsx:56]: then now you can make this a var at the top of the file

**Review by vm — APPROVED** [review]
after comment

**Pull request merged**